src/Compilers/CSharp/Portable/Binder/Semantics/Operators/OperatorKind.cs

```diff
@@ -519,6 +520,21 @@ internal enum BinaryOperatorKind
         LiftedUserDefinedRightShift = Lifted | UserDefined | RightShift,
         DynamicRightShift = Dynamic | RightShift,
 
+        IntUnsignedRightShift = Int | UnsignedRightShift,
+        UIntUnsignedRightShift = UInt | UnsignedRightShift,
+        LongUnsignedRightShift = Long | UnsignedRightShift,
+        ULongUnsignedRightShift = ULong | UnsignedRightShift,
+        NIntUnsignedRightShift = NInt | UnsignedRightShift,
+        NUIntUnsignedRightShift = NUInt | UnsignedRightShift,
+        // PROTOTYPE(UnsignedRightShift): UserDefinedUnsignedRightShift = UserDefined | UnsignedRightShift,
+        LiftedIntUnsignedRightShift = Lifted | Int | UnsignedRightShift,
+        LiftedUIntUnsignedRightShift = Lifted | UInt | UnsignedRightShift,
+        LiftedLongUnsignedRightShift = Lifted | Long | UnsignedRightShift,
+        LiftedULongUnsignedRightShift = Lifted | ULong | UnsignedRightShift,
+        LiftedNIntUnsignedRightShift = Lifted | NInt | UnsignedRightShift,
+        LiftedNUIntUnsignedRightShift = Lifted | NUInt | UnsignedRightShift,
+        LiftedUserDefinedUnsignedRightShift = Lifted | UserDefined | UnsignedRightShift,
```

Comment:
It didn't seem like this was tested, should it also be commented out with a prototype for now?

Reply:
> It didn't seem like this was tested, should it also be commented out with a prototype for now?

I do not think it is strictly necessary. Working on user-defined operators now (for the next PR)

Reply:
Alright. I don't have a problem with it, was just curious since you commented out the non-lifted variant.